### ISSUE TITLE
It's not possible to run mkmr from any dir but the root dir of the repo

### ISSUE THREAD

**Issue by Cogitri** [maxice8/mkmr#40]
This might be annoying for users which don't use `ax`.

```
Traceback (most recent call last):
  File "/home/rasmus/.local/bin/mkmr", line 4, in <module>
    main()
  File "/home/rasmus/.local/lib/python3.8/site-packages/mkmr/mkmr.py", line 136, in main
    repo = Repo()
  File "/home/rasmus/.local/lib/python3.8/site-packages/git/repo/base.py", line 181, in __init__
    raise InvalidGitRepositoryError(epath)
git.exc.InvalidGitRepositoryError: /home/rasmus/Projects/aports/main/python3
```

**Comment by maxice8** [comment]
I'll see if i can implement a repeating parent-check, so it will check if CWD has .git, then check the parent of CWD for .git then the parent of the parent of CWD for git, so on and so forth. It will go on until you get the root or the first directory without permission

**Comment by maxice8** [comment]
fixed on #48
